README.rst

```diff
@@ -1,4 +1,21 @@
 thoth-station.github.io
 -----------------------
 
-Project Thoth website
+## Project Thoth website
```

Comment:
```suggestion
Project Thoth website
================
```

In rst, we have to use the underlines for heading
https://docutils.sourceforge.io/docs/user/rst/quickstart.html#sections